packages/hint/src/lib/utils/dom/create-async-window.ts

```diff
@@ -0,0 +1,9 @@
+import { IAsyncWindow } from '../../types/async-html';
+import { JSDOMAsyncWindow } from '../../types/jsdom-async-html';
+import createJsdom from './create-jsdom';
+
+export default (html: string, allowScripts: boolean = false): IAsyncWindow => {
```

Comment:
Given my other comments about removing the `IAsync*` interfaces, can we have this return something more scoped that only provides access to evaluate scripts?